src/Application.java

```diff
@@ -87,6 +87,14 @@ private boolean commandParser(String cmd){
                 searchFriends(cmdArgs.get(1));
             }
         }
+        if(cmd.equals("rate_game")){
+            if(cmdArgs.size() != 3){
+                System.out.println("Usage: rate_game <video game title> <star rating: 1-5>");
+            } else {
+                int rating = Integer.parseInt(cmdArgs.get(2));
```

Comment:
check to make sure parseint doesn't return NaN (as in make sure the second argument was properly a number)